_layouts/article-side-menu.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+    {% include header-extended.html %}
+    <main id="main-content">
+        <img src="{{ page.hero-image }}" alt="{{ page.hero-image-alt }}" class="width-full">
+        <div class="grid-container">
+            <div class="grid-row grid-gap flex-align-start">           
+                <div class="desktop:grid-col-3 tablet:order-{{ page.side-menu-order }} margin-y-3 position-sticky top-1">
+                    <nav aria-label="Secondary navigation,">
+                        <ul class="usa-sidenav">
+                            {% for item in page.side-menu %}
+                            <li class="usa-sidenav__item">
+                                <!--Should we include the usa-current class?-->
+                                <a href="#{{ item.id }}" class="{% if item.id == "understand-the-problem" %}usa-current{% endif %}">{{ item.name }}</a>
```

Comment:
I don't there's a way to do this with Jekyll in this case because we're not linking to new pages, we're just linking to anchors on the current page. So you'd have to use some kind of JavaScript to dynamically update the HTML based on interaction with the page (e.g., clicking or scrolling).

We could just leave the static class in as an example. Probably don't need the template logic since it doesn't work in this case. If you wanted, you could play with some JS. My go-to would be `addClass` and `removeClass` with [jQuery](https://api.jquery.com/addclass/) based on clicks.